doc/manual/src/command-ref/nix-shell.md

```diff
@@ -112,11 +114,28 @@ All options not listed here are passed to `nix-store
 
 # Environment variables
 
-- `NIX_BUILD_SHELL`
+- <span id="env-NIX_BUILD_SHELL">[`NIX_BUILD_SHELL`](#env-NIX_BUILD_SHELL)</span>
 
   Shell used to start the interactive environment. Defaults to the
-  `bash` found in `<nixpkgs>`, falling back to the `bash` found in
-  `PATH` if not found.
+  `bash` from `bashInteractive` found in `<nixpkgs>`, falling back to
+  the `bash` found in `PATH` if not found.
+
+  Note that the default shell obtained using the method above may not
+  necessarily be the same as any shells requested in *path*. For
```

Comment:
What's `*path*` here, and what do you mean by request?

Reply:
> What's `*path*` here

It's the CLI argument:

https://github.com/CyberShadow/nix/blob/pull-20230811-081846/doc/manual/src/command-ref/nix-shell.md?plain=1#L16

It is pretty far away from there, maybe there is a way to refer to it less ambiguously?

> and what do you mean by request?

Indeed not sure if it is the best word bug e.g. if *path* is a file with an expression that calls `mkShell` or anything else with a specific `bash` in e.g. `buildInputs`, the goal is to communicate that that `bash` is not the `bash` that will be used by `nix-shell`.